src/record.jl

```diff
@@ -96,11 +96,13 @@ end
 end
 
 @inline function setcell!(col::Array{String,1}, i, val::StrRange, str)
+    val.escapecount>0 && error("Not yet implemented")
     col[i] = alloc_string(str, val)
     PARSE_SUCCESS
 end
 
 @inline Base.@propagate_inbounds function setcell!(col::StringVector, i, val::StrRange, str)
+    val.escapecount>0 && error("Not yet implemented")
```

Comment:
This is the place where the escaping would have to be handled for `StringArray`.